nautobot/dcim/tests/test_views.py

```diff
@@ -714,32 +714,33 @@ def test_powerports(self):
         # Test the view
         response = self.client.get(reverse("dcim:rack", args=[self.racks[0].pk]))
         self.assertHttpStatus(response, 200)
+
         # Validate Power Utilization for PowerFeed 11 is displaying correctly on Rack View.
         power_feed_11_html = """
-        <td><div title="Used: 1200&#13;Count: 3840" class="progress text-center">
+        <td><div title="Used: 1263&#13;Count: 3840" class="progress text-center">
```

Comment:
Is there any configurable mechanism for "backwards compatibility" for those who need the old numbers? Or were the old numbers unambiguously *wrong* (i.e. this is a bug fix, not merely a behavior change)?